docs/release-guide.md

```diff
@@ -1,71 +1,8 @@
 # Release process
 
-This project uses [cargo-release][cargo-release] in order to prepare new releases, to tag and sign relevant git commit and to publish the resulting artifacts to [crates.io][crates-io].
-
+This project uses [cargo-release][cargo-release] in order to prepare new releases, tag and sign the relevant git commit, and publish the resulting artifacts to [crates.io][crates-io].
 The release process follows the usual PR-and-review flow, allowing an external reviewer to have a final check before publishing.
 
-This document gives an high-level overview as well as a step-by-step guide on how to perform a release.
```

Comment:
Why did you choose to remove the high-level overview?

Reply:
I did not remove it here surgically, I just copied the whole doc from the other repo :)

I think the reason why this got removed is that now this page can be used as a checklist to live-track the release process, like https://github.com/coreos/coreos-installer/issues/297. It isn't anymore just a high-level overview.